deploy/helms/airflow/templates/web.yaml

```diff
@@ -74,15 +74,15 @@ spec:
             path: /
             port: 8080
           initialDelaySeconds: 60
-          periodSeconds: 3
-          timeoutSeconds: 2
+          periodSeconds: 5
+          timeoutSeconds: 3
```

Comment:
@aliaksandr-d what exactly the increasing of the timeouts gives us?

Reply:
Timeout was increased to 3sec, to avoid situations when kubernetes removes pod, when it's alive but currently slow due to dags number.